cross/syncthing-inotify/Makefile

```diff
@@ -0,0 +1,60 @@
+PKG_NAME = syncthing-inotify
+PKG_VERS = 0.8.4
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
```

Comment:
add  `PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)` so the dist file get's a nicer name... also have to re run `make digests`